package.json

```diff
@@ -44,9 +44,9 @@
   "homepage": "https://github.com/angular/angular-cli",
   "dependencies": {
     "@angular-cli/ast-tools": "^1.0.0",
-    "@angular/compiler": "^2.0.0-rc.6",
+    "@angular/compiler": "^2.0.0-rc.7",
```

Comment:
I actually think the compiler isn't used per se right now, but will be soon for AoT.